src/components/pages/Categories/List.vue

```diff
@@ -35,11 +41,36 @@ export default {
     access() {
       return this.$store.getters['auth/access'];
     },
+    loading() {
+      return this.$store.state.categories.loading;
+    },
+    targetCategory() {
+      return this.$store.state.categories.targetCategory.name;
+    },
+    errorMessage() {
+      return this.$store.state.categories.errorMessage;
+    },
+    doneMessage() {
+      return this.$store.state.categories.doneMessage;
+    },
   },
   created() {
     this.$store.dispatch('categories/getAllCategories');
+    this.$store.dispatch('categories/initPostCategory');
+  },
+  methods: {
+    editedCategory($event) {
```

Comment:
[nits]

`$` は Vue の組み込み関数や組み込みプロパティに使われます。
なので、それと見分けをつけるために、自分で定義した引数名などには `$` は付与しない方が良いと思います。

```suggestion
    editedCategory(event) {
```

Reply:
$を消しました